keras/layers/convolutional.py

```diff
@@ -2060,29 +2103,22 @@ class ZeroPadding1D(Layer):
     """
 
     def __init__(self, padding=1, **kwargs):
```

Comment:
Note that we could consider adding a `data_format` argument for the 1D version as well (in a later PR).